demo/src/app/components/+dropdown/demos/value-change-event/value-change-event.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-dropdown-value-change-event',
+  templateUrl: './value-change-event.html'
```

Comment:
Name `value change event` doesn't make much sense in case of dropdown because it's not supposed to have own value. It would be better to rename it to `State change event`

Reply:
also rename component name, filenames and folder names as well